ui/js/directives/treeherder/clonejobs.js

```diff
@@ -230,7 +248,19 @@ treeherder.directive('thCloneJobs', [
             jobStatus.key = getJobMapKey(job);
             jobStatus.value = job.job_type_symbol;
             jobStatus.title = getHoverText(job);
-            jobBtn = $(jobBtnInterpolator(jobStatus));
+
+            if (thResultStatus(job) === "runnable") {
+                jobStatus.buildername = job.ref_data_name;
+                jobBtn = $(runnableJobBtnInterpolator(jobStatus));
+
+                if (ThResultSetStore.isRunnableJobSelected($rootScope.repoName,
+                                                           job.result_set_id,
+                                                           jobStatus.buildername))
+                    jobBtn.addClass("runnable-job-btn-selected");
+            }
+            else{
```

Comment:
space before the {.

Sorry for all the nits, but we try to keep (are trying to get) our style consistent.  :)